contracts/TopShotLocking.cdc

```diff
@@ -0,0 +1,114 @@
+import NonFungibleToken from 0xNFTADDRESS
+
+pub contract TopShotLocking {
+
+    // -----------------------------------------------------------------------
+    // TopShotLocking contract Events
+    // -----------------------------------------------------------------------
+
+    // Emitted when a moment is withdrawn from a Collection
+    pub event Locked(id: UInt64, expiryTimestamp: UFix64)
```

Comment:
should the duration be in this also?

I would also consider adding some other identifying information to these events, such as the playID or setID.

Reply:
Since the TopShotLocking contract doesn't import the TopShot contract it makes more sense to me that it is "dumb" to all of the inner-TopShot NFT details.